pages/components/order-button/index.js

```diff
@@ -0,0 +1,11 @@
+import Link from "next/link";
+
+export default function OrderButton() {
+  return (
+    <div>
+      <button>
+        <Link href="/order-list">Order Here!</Link>
+      </button>
+    </div>
```

Comment:
You don't need the div or the button element here. In fact, a link should always just be a link. You can style it to have the "look and feel" of a button, but semantically anything that routes people somewhere is a link, and anything that does nothing but add interaction without directing people elsewhere is just a button.